Libplanet/Net/Swarm.cs

```diff
@@ -613,6 +613,8 @@ public void BroadcastTxs(IEnumerable<Transaction<T>> txs)
                     initialTip is null || !_blockChain[initialTip.Index].Equals(initialTip)
                     ? 0
                     : initialTip.Index + 1;
+                int count = 0, totalCount = _blockChain.BlockHashes.Count() - (int)initHeight;
```

Comment:
How about using `_blockChain.Count()`, if you want to count blocks?
I think it's faster if there are many blocks.
```suggestion
                int count = 0, totalCount = _blockChain.Count() - (int)initHeight;
```